### ISSUE TITLE
«puıtūa», «puıshō»

### ISSUE THREAD

**Issue by ciuak** [toaq/dictionary#52]
«puıtūa» seems broken to me. Either something is or isn't many, so to «puıtūa» would be to somehow change the speaker's perspective on x₂'s numerosity so that they think it's «puı». (Literally, «tủa hó pûı rúa» would imply that «tủa hó jîe pûı rúa na jí».) Same with «puıshō» (if not «shō»'s redundancy to «ceo»). We've already discussed this, it seems, but how shall we solve this constant variable problem?

**Comment by Ilmen-vodhr** [comment]
``___ makes property ___ be instantiated by more things.``

**Comment by uakci** [comment]
I agree with your definition, @Ilmen-vodhr, since variables should be rendered by properties. Let's see what @solpahi has to say.

**Comment by xorxes** [comment]
I disagree, but this is probably too philosophical.

Change in numerosity should not be more problematic than change in color, shape, or any other property.

"I was young and now I am old", "we were few and now we are many". "I" doesn't change referent, and neither does "we". Insert something about intensionality and extensionality here.

Also, it's in the Bible: "For this reason a man shall leave his father and his mother, and be joined to his wife; and they shall become one flesh." **jỉa shỏ gú shí**

**Comment by acotis** [comment]
I think "we" does change referent. Let's say we're talking about a book club. Previously they were four in number. Now they are eight in number. The first "they" refers to the four original members and the second "they" refers to the eight current members.

There are two possible objections. One is to say that "they" always refers to the same thing: the members of the book club. But if this is the case, then "the members of the book club" isn't the sort of semantic entity that can be the referent of a variable in toaq. IIUC, only constant values (such as the group composed of you, me, Susan, and Stevie) can be referents of variables. This would force a property reading.

The other objection is that this isn't a good example, and the **puıshō** might be satisfied as-is by a cell undergoing division, since it's basically the exact same matter that was previously one cell and is now two cells. I don't have any coherent thoughts about this objection -- it might be right.

**Comment by robintown** [comment]
This is purely a matter of eliminativism vs. coincidentalism. Toaq otherwise subscribes to coincidentalism (as eliminativism is not fit for regular human discourse), so it ought to be that pủıshō says that its argument ceases to be coincidental (mủoq) with a certain plural collection and starts being coincidental with a different plural collection of greater cardinality.

The issue, however, is that we need support for first-class plural collections for that to work. (And no, mẻ doesn't give us plural collections as individual values, it gives us mereological sums.) So if first-class plural collections are not a thing we want, the best we can do is pảqdōa—which is not ideal, as it does not allow us to talk about a book club that gained members despite losing some along the way.

**Comment by robintown** [comment]
I just added the word mẻmūoq to [gảqmēkāo](https://github.com/robintown/gaqmekao) with the definition "___ is made of ___ collectively" (again, referring not to mereological sums, but to things *coincident with plural values*). This allows us to define `pủıshō dó` quite neatly as `shỏ go jêo na pûı sa lủ mẻmūoq dó hóa`. That's all we need to properly talk about the book club's members changing, too.

**Comment by robintown** [comment]
I've just realized that @xorxes and @solpahi's intended interpretation of *pủıshō* works perfectly if we think of plural values not as constant sets, but as unary relations on individuals. The extension of such a relation would give exactly the things 'among' the plural value at any given time, and so the plural value would have no problem varying modally. Further, I believe this model would do the best job of mirroring people's intuitions about *mẻa*, as it would make phrases such as *shỏ mẻa* and *jủoq mẻa* admissible.
